datadog/threadstats/base.py

```diff
@@ -116,6 +122,13 @@ def event(self, title, text, alert_type=None, aggregation_key=None,
             'The web server is up again', alert_type='success')
         """
         if not self._disabled:
+            # Append all client level tags to every event
+            if self.constant_tags:
+                if tags:
+                    tags += self.constant_tags
+                else:
+                    tags = self.constant_tags
+
```

Comment:
`constant_tags` are not associated to events in `dogstatsd`. To be consistent, we should do the same with `ThreadStats`, what do you think ?